src/coinify/payment-method.js

```diff
@@ -110,6 +110,10 @@ PaymentMethod.fetchAll = function (inCurrency, outCurrency, api) {
   var params = {};
   if (inCurrency) { params.inCurrency = inCurrency; }
   if (outCurrency) { params.outCurrency = outCurrency; }
+  if (inCurrency === 'BTC') {
+    params.inCurrency = outCurrency;
```

Comment:
E.g. should we rename them `fetchAll(baseCurrency, quoteCurrency` ? That's probably less confusing.